src/components/button.css

```diff
@@ -0,0 +1,23 @@
+.buttonGroup {
+  display: flex;
+}
+
+.button {
```

Comment:
from the Figma design, it looks like the font type in buttons is different from the main title. I would set that here so it can be consistent in all of the buttons.